pci/src/vfio_user.rs

```diff
@@ -271,6 +271,49 @@ impl PciDevice for VfioUserPciDevice {
         self.common
             .write_bar(base, offset, data, &self.vfio_wrapper)
     }
+
+    fn move_bar(&mut self, old_base: u64, new_base: u64) -> Result<(), std::io::Error> {
+        info!("Moving BAR 0x{:x} -> 0x{:x}", old_base, new_base);
```

Comment:
We already have this information from the `detect_bar_reprogramming()` function, so maybe if you want to keep this log, it'd be worth adding `vfio-user` in there so that we understand this is related to the vfio-user device.

Reply:
The `info!()` macro includes the file and line number so we know that this is the vfio-user device.

Reply:
Haha yes true I forgot about that.